src/runner.py

```diff
@@ -3,11 +3,26 @@
 # Simple Script to replace cron for Docker
 
 import argparse
+import re
 import sys
+from datetime import datetime
 from subprocess import CalledProcessError, run
 from time import sleep, time
 
 
+def parseNumList(string):
```

Comment:
```suggestion
def parseHourList(time_str: str) -> List[int]:
```
- Nit: Maybe think about the function name just to clearly state what it's doing ...
- Lets also type annotate since bandersnatch is pretty heavily time annotated (I didn't check if suggestions are right)

Reply:
Done, I also added tests on the `parseHourList` function.
I had to call the test file as `test_docker_runner.py` in the same folder as the `runner.py` script for two reasons:

- `runner.py` is not a module, and thus the test file need to stay in the same folder to be imported correctly.
- if I called the test script as `test_runner.py` tox would be unhappy for the name confusion with the `test_runner.py` script in the repo root folder.

I hope I didn't create a mess with all these edits, wanted to contribute, but still learning how to work properly with python.